public/diagnostic.js

```diff
@@ -23,3 +23,36 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+  listOfOrders: [],
+  addToCart : Ember.computed('orders', function() {
+    let orders = Order.get('orders');
+    listOfOrders.pushObject(orders);
+  })
```

Comment:
`addToCart` is not correct. It should expect an order as an argument, and it is _that_ order which you should pass to `listOfOrders` using `pushObject` or `addObject`.

Since the `Order` Ember Class doesn't have an `orders` property, I'm not clear on what you're doing in the first line of the method.